src/arrays/ArrayUtil.java

```diff
@@ -55,41 +83,65 @@ public static int indexOf(int[] array, int value) {
      * Print new array
      */
     public static void remove(int[] array, int index) {
-
+        int[] newArray = new int[array.length - 1];
+        int num = 0;
+        for (int i = 0; i < array.length; i++) {
+            if (i != index) {
+                newArray[num++] = array[i];
+            }
+        }
+        System.out.println(Arrays.toString(newArray));
     }
 
 
     /**
      * Calculate and return sum of array's elements.
      */
     public static long sum(int[] array) {
-
-        return 0;
+        int sum = 0;
+        for (int i = 0; i < array.length; i++) {
+            sum = array[i] + sum;
+        }
+        return sum;
     }
 
 
     /**
      * Get the max value from the array.
      */
     public static int getMax(int[] array) {
-
-        return 0;
+        int max = 0;
+        for (int i = 0; i < array.length; i++) {
+            if (max < array[i]) {
+                max = array[i];
+            }
+        }
+        return max;
     }
 
     /**
      * Get the minimum value from the array
      */
     public static void getMin(int[] array) {
-
+        int min = array[0];
+        for (int i = 0; i < array.length; i++) {
+            if (min > array[i]) {
+                min = array[i];
+            }
+        }
+        System.out.println(min);
     }
 
     /**
      * Calculate average of array
      */
     public static int getAvg(int[] array) {
-
-        return 0;
+        int sum = 0;
+        int average;
+        for (int i = 0; i < array.length; i++) {
+            sum = array[i] + sum;
+        }
+        average = sum / array.length;
+        return average;
```

Comment:
done.

(for a more accurate calculation, the data can be presented in a double format)